src/generated-client/api.ts

```diff
@@ -2409,6 +2409,7 @@ export interface FlawClassification {
 }
 
 export const FlawClassificationStateEnum = {
+    NoValue: '',
```

Comment:
This is a generated file, so when it is regenerated, this change will be lost.
Is there a place in `src/types/` to update zod definitions that will work instead?

Reply:
I generated the client in a different commit so this should not be an issue anymore